crates/nu-protocol/src/errors/shell_error/mod.rs

```diff
@@ -1186,18 +1186,16 @@ This is an internal Nushell error, please file an issue https://github.com/nushe
         span: Span,
     },
 
+    /// TODO: Update help text once custom const commands are supported
```

Comment:
Minor nit, so it doesn't appear in the docs.

```suggestion
    // TODO: Update help text once custom const commands are supported
```